net-irc/znc/znc-9999.ebuild

```diff
@@ -2,163 +2,157 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
-PYTHON_COMPAT=( python3_4 )
-inherit autotools eutils python-single-r1 user
+PYTHON_COMPAT=( python{3_4,3_5} )
+PLOCALES="ru"
+CMAKE_MAKEFILE_GENERATOR=ninja
```

Comment:
Does emerge guarantee no incremental rebuild?
If it doesn't, ninja won't work reliably because only makefiles generator supports `IMPLICIT_DEPENDS`, https://github.com/znc/znc/commit/106a36c22b0a477ccb77191c50a2aadd8afa7f51